components/match2/commons/match_group_input.lua

```diff
@@ -430,4 +432,54 @@ function MatchGroupInput.readMvp(match)
 	return {players = parsedPlayers, points = mvppoints}
 end
 
+---@class MatchGroupInputReadPlayersOfTeamOptions
+---@field maxNumPlayers integer?
+---@field resolveRedirect boolean?
+---@field disallowGaps boolean?
```

Comment:
is this really needed?

Reply:
we can remove it but it would slighly adjust thefunctionality on some wikis (imo to the better)